2023/opmodes/javabotsLM1Auto.java

```diff
@@ -146,15 +150,28 @@ public void handleEvent (RobotEvent e){
                 {
                     RobotLog.i("liftedToGroundJunction");
                     //delay(1);
-                 driveToFirstGroundJunction(driveToGround1Path);
+                 driveToFromFirstGroundJunction(driveToGround1Path);
                 }
             }
         });
     }
 
-    private void driveToFirstGroundJunction(DeadReckonPath driveToGround1Path)
+    private void decideWhichSignalWasSeen() {
```

Comment:
Opening brace on a method should go on the following line.  See other examples herein.  This is a special rule for methods as in general, other brace patterns are cuddled.